velox/docs/functions/spark/string.rst

```diff
@@ -131,21 +131,20 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
 .. spark:function:: split(string, delimiter) -> array(string)
 
-    Splits ``string`` on ``delimiter`` and returns an array. ::
+    Splits ``string`` on ``delimiter`` as much as possible and returns an array. 
```

Comment:
Was the original implementation wrong? It treated 'delimiter' as a strings, but it should have treated it as regular expression.

Does Spark use Joni for regex or something else?

Reply:
Hi @mbasmanova, let me help clarify. The original version only implements the basic split version in which the pattern is a single character. So this PR will address the limitations.

I just found spark uses JDK's regex.

Reply:
In velox, split function  treated 'delimiter' as a **char**, but in spark it treated 'delimiter' as a strings and support regular expression.